### PR TITLE
Categories - Add master checkbox and Delete Selected button

### PR DESCRIPTION
**Pull request by alessandromontividiu03**
https://github.com/nopSolutions/nopCommerce/issues/4112
Added on categories page

delete-categories

### CODE REVIEW

**Comment by alessandromontividiu03** [comment]
@RomanovM

File: src/Presentation/Nop.Web/Areas/Admin/Controllers/CategoryController.cs [hunk 1]
```diff
@@ -23,6 +23,7 @@
 using Nop.Web.Framework.Controllers;
 using Nop.Web.Framework.Mvc;
 using Nop.Web.Framework.Mvc.Filters;
+using System.Collections.Generic;
```
Comment (DmitriyKulagin) [inline src/Presentation/Nop.Web/Areas/Admin/Controllers/CategoryController.cs]: Please, sort the Usings in the correct order

Comment (alessandromontividiu03) [reply src/Presentation/Nop.Web/Areas/Admin/Controllers/CategoryController.cs]: @DmitriyKulagin done.

**Comment by DmitriyKulagin** [comment]
@alessandromontividiu03 Thanks!

**Pull request merged**